unit-tests/dds/test-query-option.py

```diff
@@ -20,17 +20,26 @@
 
         with test.closure( 'Create the server' ):
             device_info.name = 'Some device'
-            s1p1 = dds.video_stream_profile( 9, dds.video_encoding.rgb, 10, 10 )
-            s1profiles = [s1p1]
             s1 = dds.color_stream_server( 's1', 'sensor' )
-            s1.init_profiles( s1profiles, 0 )
+            s1.init_profiles( [
+                dds.video_stream_profile( 9, dds.video_encoding.rgb, 10, 10 )
+                ], 0 )
             s1.init_options( [
                 dds.option( 'Backlight Compensation', dds.option_range( 0, 1, 1, 0 ), 'Backlight custom description' ),
                 dds.option( 'Custom Option', dds.option_range( 0, 1, 0.1, 0.5 ), 'Something' ),
                 dds.option( 'Option 3', dds.option_range( 0, 50, 1, 25 ), 'Something Else' )
                 ] )
+            s2 = dds.depth_stream_server( 's2', 'sensor' )
+            s2.init_profiles( [
+                dds.video_stream_profile( 27, dds.video_encoding.z16, 100, 100 )
+                ], 0 )
+            s2.init_options( [
+                dds.option( 's2 option', dds.option_range( 123, 123, 0, 123 ), 'constant' )
+                ] )
             server = dds.device_server( participant, device_info.topic_root )
-            server.init( [s1], [], {} )
+            server.init( [s1, s2], [
+                dds.option( 'IP Address', dds.option_range( 1234, 1234, 0, 1234 ), 'IP' )
```

Comment:
Notice, not a valid IP address.
Not checked now but we talked about testing by option type

Reply:
Yeah it's just a placeholder at this stage -- already changed in the next PR

Reply:
Yeah it's just a placeholder, already changed in the next PR